packages/react-slider/Spec.md

```diff
@@ -1,63 +1,232 @@
+
 - _Interaction_
   - _Keyboard_
+    | Key | Description |
+    | - | - |
+    | `Up`/`Right` | Increments the value of the slider by the `step` prop. |
+    | `Down`/`Left` | Decrements the value of the slider by the `step` prop.|
+    | `PageUp`/`Up/Right & Shift`| Increments the value of the slider by 10 _ `step`. |
+    | `PageDown` /`Down/Left & Shift`| Decrements the value of the slider by 10 _ `step`. |
+    | `Home` | Sets value to the `min` prop. |
+    | `End` | Sets value to the `max` prop. |
   - _Cursor_
+    `mousedown` sets the current value immediately.
+    `mousemove` is attached to the window element on `mousedown` and watches for move events. Value is updated accordingly.
+    `mouseup` removes the `mousemove` event.
   - _Touch_
+    Handles the same events as the _Cursor_
   - _Screen readers_
+    - **`root`:**
+      - renders `as` div
+      - handles native props expected from the element type in `as`
+    - **`thumb`:**
+      - `tabindex = 0`
+      - `role = "slider"`
+      - `aria-valuemin = min` and `aria-valuemax = max`
+      - `aria-valuenow` represents the current value.
 
 ## Accessibility
 
-Base accessibility information is included in the design document. After the spec is filled and review, outcomes from it need to be communicated to design and incorporated in the design document.
-
-- Decide whether to use **native element** or folow **ARIA** and provide reasons
-- Identify the **[ARIA](https://www.w3.org/TR/wai-aria-practices-1.2/) pattern** and, if the component is listed there, follow its specification as possible.
-- Identify accessibility **variants**, the `role` ([ARIA roles](https://www.w3.org/TR/wai-aria-1.1/#role_definitions)) of the component, its `slots` and `aria-*` props.
-- Describe the **keyboard navigation**: Tab Oder and Arrow Key Navigation. Describe any other keyboard **shortcuts** used
-- Specify texts for **state change announcements** - [ARIA live regions
-  ](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) (number of available items in dropdown, error messages, confirmations, ...)
-- Identify UI parts that appear on **hover or focus** and specify keyboard and screen reader interaction with them
-- List cases when **focus** needs to be **trapped** in sections of the UI (for dialogs and popups or for hierarchical navigation)
-- List cases when **focus** needs to be **moved programatically** (if parts of the UI are appearing/disappearing or other cases)
+`Slider` design pattern: [W3 Slider](https://www.w3.org/TR/wai-aria-1.1/#slider)
+
+## Topics for Discussion
+
+### Removing Fluid
+
+Having the `Slider` fit the width of the screen could possibly be done by the user by applying custom styling to the `Slider`.
+
+### RangedSlider Seperated
+
+It may be more readable and potentially better to split the `RangedSlider` and `Slider` components apart from each other as they have very different use cases.
```

Comment:
nit:

```suggestion
It may be more readable and potentially better to split the `RangedSlider` and `Slider` components apart from each other as they have very different use cases and accessibility concerns.
```